HackerNewsReader/Classes/AppDelegate.m

```diff
@@ -36,6 +38,13 @@ - (void)setupAppirater {
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     [self setupAppirater];
 
+    // Basic integration with Sift.
+    Sift *sift = Sift.sharedInstance;
+    sift.accountId = @"<put your account id here>";
+    sift.beaconKey = @"<put your Javascript snippet key here>";
+    sift.userId = @"<put user id here>";
```

Comment:
Hrm -- I think this would make this app unsuitable for testing some specific functionality, most notably promotion for anonymous mobile grains. So please just be sure that gets tested via some other means.